cypress/e2e/functional/opal/manualAccountCreation/addOffence/PO-411_addOffenceAdultOrYouth.feature

```diff
@@ -0,0 +1,211 @@
+Feature: PO-411 Create the 'Add an offence' screen - Adult or youth only
+
+  Background:
+    Given I am on the Opal Frontend and I sign in as "opal-test@HMCTS.NET"
+    And I am on the dashboard
+    And I navigate to Manual Account Creation
+
+    And I see "Business unit and defendant type" on the page header
+    And I enter "West London" into the business unit search box
+    And I select the "Fine" radio button
+    And I select the "Adult or youth only" radio button
+    And I click the "Continue" button
+    And I see "Account details" on the page header
+    And I click on the "Offence details" link
+    Then I see "Add an offence" on the page header
+    And I see "Offence details" text on the page
+
+  Scenario: AC1, AC2, AC3, AC4, AC5 & AC6 - New screen and fields
```

Comment:
The ticket mentions AC3 may not yet be implemented, if this is the case all good, if not it doesn't look like AC3 is covered in these tests?

Reply:
My bad I can see the steps on lines 20-34